CMakeLists.txt

```diff
@@ -26,6 +26,10 @@ if (WITH_MSAT)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DWITH_MSAT")
 endif()
 
+if (WITH_PROFILING)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DWITH_PROFILING")
```

Comment:
You could consider also checking if the library exists during configuration which gives a better warning message than it just failing to link later. It would look something like this:
```
find_library(GOOGLE_PERF profiler REQUIRED)
```

and then when you link it below you should be able to use `${GOOGLE_PERF}`.

I don't feel strongly about this though, so it's up to you.

Reply:
Thanks a lot! I just pushed the change you suggested.